KsApi/models/Backing.swift

```diff
@@ -18,7 +18,7 @@ public struct Backing {
   public let reward: Reward?
   public let rewardId: Int?
   public let sequence: Int
-  public let shippingAmount: Int?
+  public let shippingAmount: Double?
```

Comment:
Yeah I was told this was always an issue since October 2020 which must mean we simply didn't realize this was happening. This new spike must have occurred because we had a popular project that just finished funding 2-3 weeks ago.